### PR TITLE
Update command style pt2 eventing and developer directories

### PR DESCRIPTION
**Pull request by snneji**
Update the commands from using <<EOF format to a two step procedure.

Part of #3813. This PR will only changes the files in the /developer and /eventing directories.

### CODE REVIEW

File: docs/developer/eventing/sinks/kafka-sink.md [hunk 1]
```diff
@@ -151,10 +151,9 @@ in particular, [Producer configurations](https://kafka.apache.org/documentation/
 
 To enable debug logging for data plane components change the logging level to `DEBUG` in the `kafka-config-logging` ConfigMap.
 
-1. Apply the `kafka-config-logging` ConfigMap by running the command:
+1. Create a YAML file using the `kafka-config-logging` ConfigMap below:
```
Comment (abrennan89) [inline docs/developer/eventing/sinks/kafka-sink.md:154]:
```suggestion
1. Create the `kafka-config-logging` ConfigMap as a YAML file that contains the following:
```

Comment (abrennan89) [reply docs/developer/eventing/sinks/kafka-sink.md:154]: just a suggestion

File: docs/developer/eventing/sources/apiserversource/getting-started/README.md [hunk 2]
```diff
@@ -30,119 +30,146 @@ command:
...
 
-        - `<apiserversource>` is the name of the source that you want to create.
-        - `<namespace>` is the name of the namespace that you created in step 1 above.
-        - `<service-account>` is the name of the ServiceAccount that you created in step 2 above.
-        - `<sink>` is the name of the PodSpecable object that you want to use as a sink.
-        A PodSpecable is an object that describes a PodSpec.
+1. Create a ClusterRole:
 
-    === "YAML"
+    1. Create a YAML file using the template below:
```
Comment (abrennan89) [inline docs/developer/eventing/sources/apiserversource/getting-started/README.md:58]: Throughout, don't use "below". As per the Google developer style guide:

" below
    Don't use for a range of version numbers. Instead, use earlier.
    Don't use to refer to a position in a document. Instead, use later or following.
    Don't use to refer to a position in the UI. Instead, write instructions that avoid directional language. For more information, see Writing accessible documentation."

Maybe add this to the style guide also for easy reference, but it's in the [word list](https://developers.google.com/style/word-list?hl=en#letter-b)

**Review by abrennan89 — CHANGES_REQUESTED** [review]
see comments

File: docs/developer/eventing/sources/apiserversource/getting-started/README.md (hunk 2 above)
Comment (snneji) [reply docs/developer/eventing/sources/apiserversource/getting-started/README.md:58]: Sure, that's a reasonable suggestion. I'll make the changes and add this to the Knative word list.

**Comment by snneji** [comment]
@abrennan89, I've added an issue to update the style guide for the word "below" (#4074). 

I don't think I'll have time to finish doing the find and replace to change all the occurrences of  "below" before I take time off next week. If there are no other problems, can we merge this PR and I will open a new PR to apply the rule for "below"? Otherwise I fear I'll come back from my time off to some a lot of rebasing.

**Comment by abrennan89** [comment]
@snneji no worries, have a nice vacation!

**Review by abrennan89 — APPROVED** [review]
/lgtm

**Pull request merged**